models/session.js

```diff
@@ -15,7 +16,7 @@ var userSchema = new mongoose.Schema({
         type: String,
         enum: ['1', '2', '3', '5', '8', '13', '?']
     },
-    kind: {
```

Comment:
Whats the difference between this userSchema and the one in Users.js?

Reply:
Will be deleting the one in users.js since it is the same